packages/imperative/src/config/cmd/init/init.handler.ts

```diff
@@ -186,7 +186,7 @@ export default class InitHandler implements ICommandHandler {
         }
 
         this.promptProps.push(propName);
-        const propValue: any = await this.params.response.console.prompt(`Enter ${propDesc} - blank to skip: `, { hideText: property.secure });
+        const propValue: any = await this.params.response.console.prompt(`Enter ${propDesc} - Press ENTER to skip: `, { hideText: property.secure });
```

Comment:
Thanks for updating the "Press ENTER to skip" message to be consistent 🙂 Maybe we could define it as a constant somewhere? No need to change it in this PR though

Reply:
moving this to a constant...